docs/getting-started.md

```diff
@@ -0,0 +1,102 @@
+
+## Step 1. Create a catalog
+
+The catalog is what your customers are constantly working with. It is very important that the catalog is easy to browse and find the right products.
+
+Typically, to start working with the new catalog you should follow several steps:
+
+* Create a new catalogue
+* Create categories
+* Create a product
+* Create a product with variants
+
+Additionally you can read [Working with Products catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md) article. It describes how to work with products [catalog](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#common-catalogs), [categories](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#categories), [items](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#items-products), product [properties](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#properties)), product [variations](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#variations) and [virtual catalogs](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/merchandise-management/products-catalog.md#virtual-catalogs)
+
+## Step 2. Configure your store
+
+Virto Commerce platform is 4-MULTI: multi-language, multi-currency, multi-theme and multi-store by design. This gives the ability to have and run multiple stores on the same system.
+
+To configure your store you must accomplish a set of actions:
+
+1. Create first store
+2. Select your catalog
+3. Select available currency, languages
+4. Configure advanced properties:
+    1. Store description, emails and URLs
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
+
+Check that new store is visible.
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
+
+In the Virto Commerce, a price list is a set of pricing details that can be assigned to items. All prices are stored in a price list, which is associated with a catalog and a currency. When the price list is created in the Virto Commerce, you need to define which products will be included into this price list and what prices will be defined for them. You can do it by creating price list assignment.
```

Comment:
In Virto Commerce, a Pricelist is a set of pricing details that can be assigned to items. All prices are stored in the pricelist(s), which have specific currency set. When a pricelist is created, you define which products will be included in this list and what prices will be defined for them.
The way to apply the prices to products on Storefront is creating a Pricelist Assignment. It binds a catalog with pricelist and can add optional conditions for this binding.

`
Also update the initial source of this text, if there was any
`

Reply:
fixed 
In docs we use `price list` https://virtocommerce.com/docs/vc2userguide/price-lists

Reply:
price list => pricelist fixed everywhere